doc/base/classes.xml

```diff
@@ -30051,10 +30051,10 @@
 </class>
 <class name="Node2D" inherits="CanvasItem" category="Core">
 	<brief_description>
-		Base node for 2D system.
+		Barebones 2d game object, parent of all 2D related nodes. Has a position, rotation, scale and z-index.
 	</brief_description>
 	<description>
-		Base node for 2D system. Node2D contains a position, rotation and scale, which is used to position and animate. It can alternatively be used with a custom 2D transform ([Transform2D]). A tree of Node2Ds allows complex hierarchies for animation and positioning.
+		A barebones 2d game object, with a position, rotation and scale. All 2d physics nodes and sprites inherit from Node2D. Use Node2D as a parent node to move, scale and rotate children in a 2d project. Also gives control on the node's render order.
```

Comment:
I'm not a fan of "barebones". What does that really mean in this context?

 I suggest dropping it.  "A 2D game object, with a position..." seems sufficient to me.